.github/workflows/ci-linter.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        smalltalk: [ Squeak64-5.2 ]
+        smalltalk: [ Squeak64-Trunk ]
```

Comment:
We had trunk as the default target as well, but as trunk is not stable enough for a ci (we had to merge without trunk approval quite often) we opted for a stable version.

Reply:
Fair point. Resolved via 3a6cd1f.